scripts/exglobal_land_analysis_run.sh

```diff
@@ -0,0 +1,175 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_run.sh
+#
+# This script runs a global land letkfoi analysis with FV3-JEDI.
+# It assumes the runtime directory has already been staged with the appropriate
+# input files and YAML configuration (by the initialize script) before execution.
+#
+################################################################################
+#  Utilities
+export NCP=${NCP:-"/bin/cp"}
+export NMV=${NMV:-"/bin/mv"}
+export NLN=${NLN:-"/bin/ln -sf"}
+CASE=${CASE:-C96}
+RES=$(echo "${CASE}" | cut -c 2-)
```

Comment:
All but `RES` are defined in config.base and don't really need to be redefined here. The `RES` definition can be written without using cut:
```suggestion
RES=${CASE:1}
```

Reply:
Made changes as suggested.